tests/model/write_model_test.go

```diff
@@ -0,0 +1,427 @@
+package model
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"testing"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+	parser "github.com/craigpastro/openfga-dsl-parser/v2"
+	"github.com/openfga/openfga/cmd"
+	"github.com/openfga/openfga/pkg/typesystem"
+	"github.com/stretchr/testify/require"
+	pb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	healthv1pb "google.golang.org/grpc/health/grpc_health_v1"
+	"google.golang.org/grpc/status"
+)
+
+var tests = map[string]struct {
+	model string
+	code  int
+}{
+	//	"case1": {
```

Comment:
Can we remove commented out code and/or wire up these parse error tests?

Reply:
Removed for now. Thanks!